### ISSUE TITLE
Abandoned?

### ISSUE THREAD

**Issue by fredgolightly** [kollerlukas/Camera-Roll-Android-App#248]
I'm wondering if this project has been abandoned, and if so if, as it's open source, someone else would maintain it?

**Comment by kollerlukas** [comment]
Oh man, I don't know.
I have kind of lost the interest in this project. There are a lot of things I would like to change, most importantly re-coding everything in Kotlin. But this is all a lot of work.
On the other I can't maintain this app forever.
I am still debating with myself what to do. 
But at the moment I am tending towards focusing on to new projects.

Anyone who wants to continue to work on this project is happily invited to.

Sorry!

**Comment by fredgolightly** [comment]
Damn, that's a real shame. It's a super fast app and I use it over every other gallery app on all my devices. Would it be ok to post it on xda games & apps section & say it's potentially available for someone else to carry it on?

**Comment by eauchat** [comment]
I totally agree with @fredgolightly Camera-Roll is really a great and it would be a shame if it's discontinued.
I had been looking for so long for a good gallery app and I'm a real fan of it. I would take on the project myself if I had the competences.
Thanks really a lot for the great work you did all this time @kollerlukas.
I really hope this will be continued somehow :)

**Comment by nvi9** [comment]
I _might_ have the competences, but I'm lack of time until mid-summer. I've created a [PR] (#244) for Christmas, and since made other updates on it too (but not pushed yet), mainly fixes.

**Comment by fredgolightly** [comment]
Wonderful! I'm not sure how Play Store publishing would work but I'm now looking forward to summer!

**Comment by IzzySoft** [comment]
I don#t care about Play Store as long as F-Droid packages new releases – but if Lukas would be willing to make the releases for Play, that shouldn't be an issue.

**Comment by marlid91** [comment]
Your app is really great. The best designed gallery app I've found.

**Comment by Myself5** [comment]
I wouldn't mind helping out in case we come across any issues on @CarbonROM.

I can confirm #244 is needed, I just did the same fix because I wanted to look into #188 which is still existing on the latest release version.

Is it possible to release a new build that includes 5dbb0e7d1a8934b0988ecdb36d220faeeac8e203 as well as #244 (to fix the build)?

**Comment by rancidfrog** [comment]
This is by far the best gallery app.
I hope development continues, in the same way.
Simple, clean.

**Comment by fredgolightly** [comment]
@Myself5 Are you able to maintain this via a fork?

**Comment by Myself5** [comment]
@fredgolightly we've moved to Simple Gallery due to a mixture of missing features (mainly an editor) and personal taste.
